src/index.ts

```diff
@@ -3,7 +3,6 @@ export * from "./auth";
 export * from "./errors";
 export * from "./expressServer";
 export * from "./logger";
-export * from "./passport";
```

Comment:
What were the modifications to the vendored passport vs the library?

Reply:
Not much! I think mainly adding token to our user for ease of transport (which we weren't actually using anyway since we were using async/secure storage!).